packages/next-preact/index.js

```diff
@@ -12,9 +14,9 @@ module.exports = (nextConfig = {}) => {
       }
 
       config.resolve.alias = Object.assign({}, config.resolve.alias, {
-        react$: 'preact-compat',
+        react$: path.resolve(__dirname, './preact-compat.js'),
```

Comment:
This should likely be `join` as well.
```suggestion
        react$: path.join(__dirname, './preact-compat.js'),
```